package.json

```diff
@@ -340,7 +340,9 @@
     "onCommand:csharp.downloadDebugger",
     "onCommand:csharp.listProcess",
     "onCommand:csharp.listRemoteProcess",
-    "workspaceContains:project.json"
+    "workspaceContains:project.json",
+    "workspaceContains:*.csproj",
```

Comment:
I was thinking of scenarios like the one described by https://github.com/OmniSharp/omnisharp-vscode/issues/1328.